ui/packages/ui/src/main/webapp/components/installer-navigation/installer-navigation-buttons.hbs

```diff
@@ -43,10 +43,10 @@
 {{else}}
     <li>
         {{#if busy}}
-            <a id="shutdownNowStep" class="disabled-next" href="#">Shutdown Now</a>
+            <a id="shutdownNowStep" class="disabled-next" href="#">Restart Later</a>
```

Comment:
Unless the keystores are updated, there is no longer a reason to restart DDF after installation.

Reply:
Excellent.